src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -132,6 +132,27 @@ class CoreWorkerDirectActorTaskSubmitter
   /// Check timeout tasks that are waiting for Death info.
   void CheckTimeoutTasks();
 
+  /// Block the actor_task_io_thread_ until the submitter is shutdown.
+  void WaitForShutdown();
+
+  void Shutdown();
+
+  /// Push a task to corresponding client queue by the actor id.
+  ///
+  /// \param[in] task The task spec to push.
+  /// \param[in] callback  The callback function to be called after the task spec be sent
```

Comment:
can we create a private helper function for this? exposing test functionality in public method is a bad practice.

Reply:
`AppendTask` removed.